org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/RenameService2.xtend

```diff
@@ -197,11 +200,11 @@ class RenameService2 implements IRenameService2 {
 					if (element !== null && !element.eIsProxy) {
 						val leaf = NodeModelUtils.findLeafNodeAtOffset(rootNode, candidateOffset)
 						if (leaf !== null && leaf.isIdentifier) {
-							val leafText = NodeModelUtils.getTokenText(leaf)
+							val leafText = getConvertedValue(leaf.grammarElement, leaf)
```

Comment:
Given that this is already quite a bit of complexity in the logic, please change the variable name to better reflect what it is now.

Reply:
Done_